config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml

```diff
@@ -81,3 +81,40 @@ presets:
       - key: capv-gcs-keyfile.json
         path: keyfile.json
         mode: 288
+- labels:
+    preset-image-builder-e2e-config: "true"
```

Comment:
I think in the preset we should add vsphere in some way (as image-builder is not only vSphere)

Maybe `preset-image-builder-vsphere-e2e-config`

(Just to "namespace" the preset a bit within test-infra)